project_name/project_name/celery.py

```diff
@@ -0,0 +1,22 @@
+"""
+Celery config for {{ project_name }} project.
+
+For more information on this file, see
+http://celery.readthedocs.org/en/latest/django/first-steps-with-django.html
+
+Run your celery worker(s) as `celery -A {{ project_name }} worker --loglevel=info`
+"""
+
+from __future__ import absolute_import
+
+import os
+from celery import Celery
+from django.conf import settings
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "{{ project_name }}.settings.production")
```

Comment:
Could we set the default settings file to dev settings like manage.py? (production servers would set the ``DJANGO_SETTINGS_MODULE`` variable to make it use production settings anyway)